src/simple.jl

```diff
@@ -97,8 +97,13 @@ end
 return: a solution matrix with time steps in `t` along rows and
         state variable `y` along columns
 """
-function cvode(f::Function, y0::Vector{Float64}, t::Vector{Float64}, userdata::Any=nothing;
-              integrator=:BDF, reltol::Float64=1e-3, abstol::Float64=1e-6)
+cvode(f::Function, y0::Vector{Float64}, t::AbstractVector, userdata::Any=nothing;
+                integrator=:BDF, reltol::Float64=1e-3, abstol::Float64=1e-6) = 
+    cvode!(f, y0, t, zeros(length(t), length(y0)), userdata; 
+                    integrator=integrator, reltol=reltol, abstol=abstol)
+
+function cvode!(f::Function, y0::Vector{Float64}, t::AbstractVector, y::Matrix{Float64}, userdata::Any=nothing;
```

Comment:
I think this is the only remaining problem before merging. Is the author still around for this? Or should I just merge and make this quick change?